tasks/worker/filesystem.yml

```diff
@@ -7,6 +7,13 @@
   become: yes
   when: concourse_baggageclaim_driver == 'btrfs'
 
+- name: Check btrfs-progs is present
```

Comment:
Nit: this playbook uses all lower case letters for task names.
Nit: I think "install btrfs-progs package" may be more informative.